test/RPCSession.cpp

```diff
@@ -217,10 +217,10 @@ void RPCSession::test_setChainParams(vector<string> const& _accounts)
 	{
 		"sealEngine": "NoProof",
 		"params": {
-			"accountStartNonce": "0x",
+			"accountStartNonce": "0x00",
 			"maximumExtraDataSize": "0x1000000",
 			"blockReward": "0x",
-			"allowFutureBlocks": "1",
+			"allowFutureBlocks": true,
```

Comment:
I did not change that on purpose to be compatible with both the old and the new version of eth. Let's see what the tests say.